task/sast-unicode-check/0.1/README.md

```diff
@@ -0,0 +1,30 @@
+# sast-unicode-check task
+
+## Description:
+
+The sast-unicode-check task uses [find-unicode-control](https://github.com/siddhesh/find-unicode-control.git)  tool to perform Static Application Security Testing (SAST) to look for non-printable unicode characters in all text files in a source tree.
+
+## Parameters:
+
+| name                         | description                                                                                      |
+|------------------------------|--------------------------------------------------------------------------------------------------|
+| FIND_UNICODE_CONTROL_GIT_URL | URL from repository to find unicode control                                                      |
+| KFP_GIT_URL                  | Link to the known-false-positives repository. If left blank, results won't be filtered           |
```

Comment:
Please change the description to `Known False Positives git URL, optionally taking a revision delimited by #; If empty, filtering of known false positives is disabled.` to make it consistent with #1411.

Reply:
Updated